lib/everest/slac/io/include/everest/slac/io.hpp

```diff
@@ -20,6 +20,9 @@ class SlacIO {
     void send(slac::messages::HomeplugMessage& msg);
     void quit();
 
+    // cannot be const while libslac's SlacChannel::get_mac_addr() isn't const
```

Comment:
The comment says it cannot be const but then it is defined as const?

Reply:
I meant a const function, not a const return parameter.

Because `Channel::get_mac_addr()` isn't const in https://github.com/EVerest/libslac/blob/v0.3.1/include/slac/channel.hpp#L43:
`const uint8_t* get_mac_addr();`
instead of
`const uint8_t* get_mac_addr() const;`

this method in SlacIO must use
`const uint8_t* get_mac_addr() /* const */;`
instead of the preferred
`const uint8_t* get_mac_addr() const;`

Reply:
understood